lib/nrf91_lwm2m_app/Kconfig

```diff
@@ -4,66 +4,12 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-menuconfig NRF91_LWM2M_APP
+config NRF91_LWM2M_APP
```

Comment:
Why is this no longer a `menuconfig` ?
Now all config settings is expanded into current menu, instead of having a menu of their own like: `LTE_LINK_CONTROL`, `NRF_MODEM_LIB`, etc. has.

Reply:
Fixed.